app/components/cloud-call.js

```diff
@@ -110,7 +94,6 @@ export default Component.extend({
       if (!shouldRender) return;
```

Comment:
FYI this only works when its the only instance of the component on the whole app, you'd need to ember-ize every function otherwise